pages/linux/dockerd.md

```diff
@@ -0,0 +1,24 @@
+# dockerd
+
+> A persistent process to start and manage docker containers.
+> More information: <https://docs.docker.com/engine/reference/commandline/dockerd/>.
+
+- Run docker daemon:
+
+`dockerd`
+
+- Run docker daemon and config it to listen to specific sockets(unix,tcp):
+
+`dockerd --host unix://{{path/to/*.sock}} --host tcp://{{IP}}`
```

Comment:
Let's use an actual file. For eg. {{path/to/tmp.sock}}. Also let's change IP to ip.

Reply:
- Could you please explain why do you want to token name to be some specific file instead of *.sock when it can be any sock file?
- I already change from IP > ip.

Reply:
Please see https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md#token-syntax (last point). In short, a token already signifies a variable value, something that the user should change when running the real command. path/to/*.sock is not a valid file path. The concept that it can be any file is already expressed by putting the value in tokens (`{{}}`). After that we can choose any specific value to show what it can be. That's the reason to choose tmp.sock.

Reply:
Done.